Userland/Libraries/LibJS/Runtime/DateConstructor.cpp

```diff
@@ -156,26 +158,15 @@ DateConstructor::~DateConstructor()
 {
 }
 
-struct DatetimeAndMilliseconds {
-    Core::DateTime datetime;
-    i16 milliseconds { 0 };
-};
-
-static DatetimeAndMilliseconds now()
-{
-    struct timeval tv;
-    gettimeofday(&tv, nullptr);
-    auto datetime = Core::DateTime::now();
-    auto milliseconds = static_cast<i16>(tv.tv_usec / 1000);
-    return { datetime, milliseconds };
-}
-
 // 21.4.2.1 Date ( ...values ), https://tc39.es/ecma262/#sec-date
 ThrowCompletionOr<Value> DateConstructor::call()
 {
-    auto [datetime, milliseconds] = JS::now();
-    auto* date = Date::create(global_object(), datetime, milliseconds, false);
-    return js_string(heap(), date->string());
+    // 1. If NewTarget is undefined, then
+    //     a. Let now be the time value (UTC) identifying the current time.
+    auto now = AK::Time::now_realtime().to_milliseconds();
```

Comment:
The LHS was more explicit about this being in UTC. Maybe the AK method should be called now_realtime_utc() if it always returns utc, or there should be a new now_utc() if we don't want to limit realtime_ to always return utc?

(I'm aware that serenity is currently always utc as system time, but that'll change at some point. Then again, maybe Time objects will always be utc internally even then.)

Mostly I'm saying the LHS used Apis that are well known to use utc, and it's less obvious now (to me, at least).

Reply:
Yeah it makes sense to include "utc" in that function name. We use the `CLOCK_REALTIME` for `now_realtime`, which gives seconds since the Epoch (which POSIX defines as UTC), so `now_realtime` will be UTC time. Calling it `now_utc` would certainly be clearer.